com.chartboost.mediation/Runtime/Plugins/iOS/ChartboostMediationUnityBridge.mm

```diff
@@ -665,13 +665,13 @@ void _chartboostMediationDiscardOversizedAds(BOOL shouldDiscard)
         NSString * partnerDisplayNameKey    = @"partnerDisplayName";  NSString * partnerDisplayNameValue = adapter.partnerDisplayName;
 
         NSDictionary *adapterDictionary = [NSDictionary dictionaryWithObjectsAndKeys:
-          partnerVersionValue,partnerVersionKey,
-          adapterVersionValue,adapterVersionKey,
-          partnerIdentifierValue,partnerIdentifierKey,
-          partnerDisplayNameValue,partnerDisplayNameKey,
-          nil
+                                               partnerVersionValue,partnerVersionKey,
+                                           adapterVersionValue,adapterVersionKey,
+                                           partnerIdentifierValue,partnerIdentifierKey,
+                                           partnerDisplayNameValue,partnerDisplayNameKey,
+                                           nil
```

Comment:
Accidental formatting by the IDE? 🤔

Reply:
Proper formatting by the IDE. This is the final spacing after formatted by XCode, if you copy paste into Rider it will mess it up, if you paste into VS Code it will keep the proper whitespace.